client/js/components/document/DpMapSettingsPreview.vue

```diff
@@ -73,10 +73,10 @@
             :data-cy="Translator.trans(link.label)"
             :href="href(link)"
             class="o-link"
-            :class="{'color--system-confirm':link.done()}">
+            :class="{'color-ui-status-complete-text': link.done()}">
```

Comment:
As I wrote in https://github.com/demos-europe/demosplan-ui/pull/108 I personally would omit the `ui-`

Reply:
Changed in https://github.com/demos-europe/demosplan-core/pull/1326